index.html

```diff
@@ -0,0 +1 @@
+Hello there, awesome person!
```

Comment:
<html>
<body>
```suggestion
Hello there, awesome person!
```
</body>
</html>